taoswswrap/tdengine_connection.py

```diff
@@ -56,7 +57,7 @@ def __init__(self, tb, err):
         self.err = err
 
 
-def values_to_column(values, column_type):
+def values_to_column(values: list, column_type):
```

Comment:
just to keep consistency 
```suggestion
def values_to_column(values: list, column_type: str):
```